showcase/image_analysis.md

```diff
@@ -0,0 +1,57 @@
+
+
+## What can you use the tool for?
+ 
+The Mouse Brain Alignment Tool provides an automated pipeline for analyzing brain autoradiography images. MBAT allows end-users to focus on image analysis without technical interruption, thereby increasing the reliability and reproducibility of results, thus expanding the possibilities of neurobiology research. The software is open source and packaged so that anyone, even without administrator access, can download and readily use the tool.
+The current iteration of MBAT is only applicable to ARG images of mouse brain, as it relies on the Mouse Brain Atlas. However, by incorporating additional standardized brain atlases, such as those for the rat animal model, it could be made more widely applicable to other brain analyses. Currently, it is only compatible with ARG scans; however, with minor modifications, ARG images set up on whole slides could be accommodated. With a few minor modifications, the tool can also be applied to a range of different image file types. In essence, the tool is useful for any application that aims to measure signals in coronal slices of the brain, whether through autoradiography or alternative staining methods, including fluorescent staining. Thus, the tool can greatly facilitate neuroinflammation research, whether induced through infectious diseases or other causes.
+
+## Resources
+
+1: Guedj, E et al. “18F-FDG brain PET hypometabolism in patients with long COVID.” European journal of nuclear medicine and molecular imaging vol. 48,9 (2021): 2823-2833. doi:10.1007/s00259-021-05215-4
+2: https://www.medrxiv.org/content/10.1101/2022.06.02.22275916v1 
+
+News-article: https://www.eurobioimaging.eu/news/an-open-source-image-analysis-tool-for-studying-neuroinflammation-linked-to-covid-19/
+
+MBAT on Github: https://github.com/Turku-BioImaging/mouse-brain-alignment-tool 
```

Comment:
Since MBAT is linked in the resource table, this is redundant imo

Reply:
done